[<Onnx] onnx.pb.h not created as part of build

### Operating system

Windows

### Compiler

Visual Studio 2022 Commuity

### Steps to reproduce the behavior

```Shell
vcpkg install onnx --recurse

In a program, include the file

#include <onnx/onnx_pb.h>
```


### Failure logs

Compile
```
Build started at 2:08 PM...
1>------ Build started: Project: test, Configuration: Debug x64 ------
1>main.cpp
1>c:\vcpkg\installed\x64-windows\include\onnx\onnx_pb.h(53,10): error C1083: Cannot open include file: 'onnx/onnx.pb.h': No such file or directory

```

Note that there are two '.' (periods) in the file name.


### Additional context

_No response_

--- a/ports/onnx/portfile.cmake
+++ b/ports/onnx/portfile.cmake
@@ -64,6 +64,10 @@ vcpkg_cmake_install()
 vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/ONNX)
 
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/${PORT}/ONNXConfig.cmake" "# import targets" 
+[[# import targets
+include(CMakeFindDependencyMacro)
+find_dependency(protobuf CONFIG)]])
 
 file(REMOVE_RECURSE
     "${CURRENT_PACKAGES_DIR}/debug/include"

--- a/ports/onnx/vcpkg.json
+++ b/ports/onnx/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "onnx",
   "version-semver": "1.15.0",
+  "port-version": 1,
   "description": "Open standard for machine learning interoperability",
   "homepage": "https://onnx.ai",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6322,7 +6322,7 @@
     },
     "onnx": {
       "baseline": "1.15.0",
-      "port-version": 0
+      "port-version": 1
     },
     "onnx-optimizer": {
       "baseline": "0.3.18",

--- a/versions/o-/onnx.json
+++ b/versions/o-/onnx.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "69ad297b15bb801f74d50be6e5d290eab7573be4",
+      "version-semver": "1.15.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "c143b899cbe396d82182568746203a3375fc23df",
       "version-semver": "1.15.0",
       "port-version": 0

